circle.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - checkout
       - run:
           name: "Update submodules"
-          command: git submodule update --init
+          command: git submodule update --init --recursive
```

Comment:
You know https://github.com/chfast/intx/issues/68 would solve this :wink:

Reply:
Right, maybe will do it before first intx release.